features/commands/install.feature

```diff
@@ -634,3 +634,24 @@ Feature: berks install
     """
     Resolving cookbook dependencies...
     """
+
+  Scenario: when using a chef_repo source
+    Given I use a fixture named "complex-cookbook-path"
+    And a file named "Berksfile" with:
+      """
+      source chef_repo: '.'
+
+      cookbook 'jenkins-config'
+      """
+    When I successfully run `berks install`
+    Then the output should contain:
+      """
+      Installing jenkins (2.0.1) from
```

Comment:
It prints the absolute path since that's more useful for debugging I think, and I didn't feel like tracking down what that would be in Aruba :)